config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :seances
+  resources :movies
+  resources :halls
```

Comment:
Since you have a relation between `seances` and `movies` and so on, you can create nested routes like this:
```ruby
resources :movies do
  resources :seances
end
```

Thanks for that, you can get all seances for some movie `seances/1/movies`